test/parallel/test-os.js

```diff
@@ -85,9 +85,12 @@ const hostname = os.hostname();
 is.string(hostname);
 assert.ok(hostname.length > 0);
 
-const uptime = os.uptime();
-is.number(uptime);
-assert.ok(uptime > 0);
+// On IBMi, `os.uptime()` is undefined
```

Comment:
This comment and the one on line 250 are wrong, aren't they? `os.uptime()` exists but it _returns_ `undefined`.

Reply:
Yes, it returns `undefined`, shall I change it to `On IBMi, 'os.uptime()' returns 'undefined'`?